test/graph/GraphQLTest.ts

```diff
@@ -58,4 +58,18 @@ query Repos($teamId: ID!, $offset: Int!) {
         const errors = GraphQL.validateQuery(query);
         assert(errors.length === 0);
     });
+
+    it("should successfully load query from relative path and replace parameters in subscription", () => {
+        const query = GraphQL.subscriptionFromFile(
+            "./someSubscription",
+            __dirname,
+            {
+                teamId: "T1L0VDKJP",
+                offset: 100,
+                isPrivate: true,
+            });
+        assert(query.includes(`"T1L0VDKJP"`));
+        assert(query.includes("100"));
+        assert(query.includes("private: true"));
```

Comment:
The query is small enough that I'd have the properly replaced version as a string and assert ===.

Reply:
It's probably worth adding tests for multiple replacements of a single variable and the `$foo`/`$fooBar` case.

Reply:
Maybe you can come up with some regexp trickery. But for now I'm not too concerned about that problem.